### PR TITLE
PIL-1182: added exchange fee capture

### PR DESCRIPTION
**Pull request by poocart**
https://linear.app/pillarproject/issue/PIL-1182/fee-capture

### CODE REVIEW

File: src/screens/Exchange/Exchange/Exchange.js [hunk 1]
```diff
@@ -129,7 +137,19 @@ function Exchange() {
     setToAddress(asset.address);
   };
 
-  const handleOfferPress = (offer: ExchangeOffer) => {
+  const handleOfferPress = async (selectedOffer: ExchangeOffer) => {
+    if (!activeAccount) {
```
[the remark this answers is not in the document]
Comment (poocart) [reply src/screens/Exchange/Exchange/Exchange.js:141]: Yes, this shouldn't happen, but in the models we have active account as optional and `flow` fail if we treat in a different way, hope mot makes sense.

[the remark this answers is not in the document]
Comment (IAmKio) [reply src/screens/Exchange/Exchange/Exchange.js:141]: Got it - thanks for that!

**Comment by IAmKio** [comment]
@poocart FYI all values added to Remote Config.

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**